pkg/datamodels/fiware/beach.go

```diff
@@ -9,7 +9,7 @@ import (
 	ngsi "github.com/diwise/ngsi-ld-golang/pkg/ngsi-ld/types"
 )
 
-//Beach is a Fiware entity
+// Beach is a Fiware entity
 type Beach struct {
```

Comment:
ngsi-ld-golang vs. https://github.com/diwise/context-broker/tree/main/pkg/datamodels/fiware?

Reply:
I'm actually not sure which one I should be preferring but perhaps it makes more sense to migrate to using context-brokers models?